src/utils/fusionAuth.ts

```diff
@@ -0,0 +1,150 @@
+import fs from 'fs-extra';
+import os from 'os';
+import path from 'path';
+import prompts from 'prompts';
+import Webhead, { WebheadInstance } from 'webhead';
+
+const builderApiURL = '{HOST}/api/builder';
+const fusionAuthURL = 'https://fusionauth-ce.betty.services';
+
+type response = string | object | null;
```

Comment:
Types should start with a capital

Reply:
`object` is about the same as `any` 😁 Is it possible to get the correct type from somewhere or create a proper type?